backend/project_alpha/api/serializers.py

```diff
@@ -50,6 +50,13 @@ class Meta:
         model = User
         fields = ('nickname',)
 
+class ChangeEmailSerializer(serializers.ModelSerializer):
```

Comment:
Let's try the next thing:
 - Make this serializer _not_ ModelSerializer, but just regular
 - Add confirm_password field into seraizlier
 - Move validation logic to view
 - As uniq email constraint is defied in model, I think you can wrap user update call into try/catch without explicit check for mail